src/input/pointer/mod.rs

```diff
@@ -409,6 +409,12 @@ impl<D: SeatHandler + 'static> PointerHandle<D> {
         self.inner.lock().unwrap().location
     }
 
+    /// Access the [`Serial`] of the last `pointer_enter` event.
+    #[cfg(feature = "wayland_frontend")]
+    pub fn last_enter(&self) -> Option<Serial> {
```

Comment:
Could we mention in the docs that this is cleared on leave. 
Just from the signature and description I did not expect last_enter to be affected by leave.

Reply:
done!